== SUBMITION (2019-06-12 13:08:23) 
-- CODE:
from math import*
from numpy import*
x = array(eval(input("valores")))
n = size(x)
m = sum(x/n)
s = 0
for i in range(n):
	s = s + (x[i] - m)**2
d = ((s / (n-1))**0.5)
print(roun(d,3))
-- EXECUTION TIME:
1.0663
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1798/2569/main.py", line 10, in <module>
    print(roun(d,3))
NameError: name 'roun' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-12 13:08:39) 
-- CODE:
from math import*
from numpy import*
x = array(eval(input("valores")))
n = size(x)
m = sum(x/n)
s = 0
for i in range(n):
	s = s + (x[i] - m)**2
d = ((s / (n-1))**0.5)
print(roun(d,3))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 2, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-12 13:08:55) 
-- CODE:
from math import*
from numpy import*
x = array(eval(input("valores")))
n = size(x)
m = sum(x/n)
s = 0
for i in range(n):
	s = s + (x[i] - m)**2
d = ((s / (n-1))**0.5)
print(roun(d,3))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 2, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-12 13:08:56) 
-- CODE:
from math import*
from numpy import*
x = array(eval(input("valores")))
n = size(x)
m = sum(x/n)
s = 0
for i in range(n):
	s = s + (x[i] - m)**2
d = ((s / (n-1))**0.5)
print(roun(d,3))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 2, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-12 13:08:56) 
-- CODE:
from math import*
from numpy import*
x = array(eval(input("valores")))
n = size(x)
m = sum(x/n)
s = 0
for i in range(n):
	s = s + (x[i] - m)**2
d = ((s / (n-1))**0.5)
print(roun(d,3))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 2, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-12 13:08:56) 
-- CODE:
from math import*
from numpy import*
x = array(eval(input("valores")))
n = size(x)
m = sum(x/n)
s = 0
for i in range(n):
	s = s + (x[i] - m)**2
d = ((s / (n-1))**0.5)
print(roun(d,3))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 2, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-12 13:09:16) 
-- CODE:
from math import*
from numpy import*
x = array(eval(input("valores")))
n = size(x)
m = sum(x/n)
s = 0
for i in range(n):
	s = s + (x[i] - m)**2
d = ((s / (n-1))**0.5)
	print(roun(d,3))
-- ERROR:
File "XXXX", line 10
    print(roun(d,3))
    ^
IndentationError: unexpected indent
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-12 13:09:17) 
-- CODE:
from math import*
from numpy import*
x = array(eval(input("valores")))
n = size(x)
m = sum(x/n)
s = 0
for i in range(n):
	s = s + (x[i] - m)**2
d = ((s / (n-1))**0.5)
	print(roun(d,3))
-- ERROR:
File "XXXX", line 10
    print(roun(d,3))
    ^
IndentationError: unexpected indent
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-12 13:09:17) 
-- CODE:
from math import*
from numpy import*
x = array(eval(input("valores")))
n = size(x)
m = sum(x/n)
s = 0
for i in range(n):
	s = s + (x[i] - m)**2
d = ((s / (n-1))**0.5)
	print(roun(d,3))
-- ERROR:
File "XXXX", line 10
    print(roun(d,3))
    ^
IndentationError: unexpected indent
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-12 13:09:17) 
-- CODE:
from math import*
from numpy import*
x = array(eval(input("valores")))
n = size(x)
m = sum(x/n)
s = 0
for i in range(n):
	s = s + (x[i] - m)**2
d = ((s / (n-1))**0.5)
	print(roun(d,3))
-- ERROR:
File "XXXX", line 10
    print(roun(d,3))
    ^
IndentationError: unexpected indent
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-06-12 13:09:19) 
-- CODE:
from math import*
from numpy import*
x = array(eval(input("valores")))
n = size(x)
m = sum(x/n)
s = 0
for i in range(n):
	s = s + (x[i] - m)**2
d = ((s / (n-1))**0.5)
	print(roun(d,3))
-- EXECUTION TIME:
1.03622
-- ERROR:
  File "/home/codebench/codes/1798/2569/main.py", line 10
    print(roun(d,3))
    ^
IndentationError: unexpected indent

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-06-12 13:09:39) 
-- CODE:
from math import*
from numpy import*
x = array(eval(input("valores")))
n = size(x)
m = sum(x/n)
s = 0
for i in range(n):
	s = s + (x[i] - m)**2
	d = ((s / (n-1))**0.5)
print(roun(d,3))
-- EXECUTION TIME:
0.937733
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1798/2569/main.py", line 10, in <module>
    print(roun(d,3))
NameError: name 'roun' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-06-12 13:09:44) 
-- CODE:
from math import*
from numpy import*
x = array(eval(input("valores")))
n = size(x)
m = sum(x/n)
s = 0
for i in range(n):
	s = s + (x[i] - m)**2
	d = ((s / (n-1))**0.5)
print(roun(d,3))
-- EXECUTION TIME:
0.827619
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1798/2569/main.py", line 10, in <module>
    print(roun(d,3))
NameError: name 'roun' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-06-12 13:09:48) 
-- CODE:
from math import*
from numpy import*
x = array(eval(input("valores")))
n = size(x)
m = sum(x/n)
s = 0
for i in range(n):
	s = s + (x[i] - m)**2
	d = ((s / (n-1))**0.5)
	print(roun(d,3))
-- EXECUTION TIME:
1.11267
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1798/2569/main.py", line 10, in <module>
    print(roun(d,3))
NameError: name 'roun' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-12 13:10:17) 
-- CODE:
from math import*
from numpy import*
x = array(eval(input("valores")))
n = size(x)
m = sum(x/n)
s = 0
for i in range(n):
	s = s + (x[i] - m)**2
d = ((s / (n-1))**0.5)
print(roun(d,3))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 2, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-06-12 13:10:40) 
-- CODE:
from math import*
from numpy import*
x = array(eval(input("valores")))
n = size(x)
m = sum(x/n)
s = 0
for i in range(n):
	s = s + (x[i] - m)**2
d = ((s / (n-1))**0.5)
print(round(d,3))
-- EXECUTION TIME:
0.92936
-- TEST CASE 1:
---- input:
[-4,250,-1,30,12]
---- correct output:
108.498
---- user output:
108.498
-- TEST CASE 2:
---- input:
[12.5,25.2,50.39,47.3,0.9]
---- correct output:
21.526
---- user output:
21.526
-- TEST CASE 3:
---- input:
[1.5,1.9,1.2,1.3,1.4,1.1,1.75]

---- correct output:
0.29
---- user output:
0.29
-- GRADE:
100%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
